gluefactory/models/lines/pold2_extractor.py

```diff
@@ -355,9 +376,14 @@ def test_extractor(extractor, folder_path, device, show=False):
     # Start counter
     start_time = time.perf_counter()
 
-    data = {"points": points, "distance_map": distance_map, "angle_map": angle_map}
+    data = {
+        "points": points,
+        "distance_map": distance_map,
+        "angle_map": angle_map,
+        "descriptors": torch.zeros(len(points), 128).to(device)
```

Comment:
I would move the descriptor dimension into a const Variable and referenc eto it here. Makes it more clear and configurable

Reply:
Definitely. But this was just to test line extraction which would expect descriptors of the shape [B,N,D] where D can be anything, and we don't use the descriptors here except indexing them in the end after filtered line endpoints after NMS. I would assume the descriptor dimension is set in the extractor config (JPLDD).